internal/platform/gin/auditlog/middleware_test.go

```diff
@@ -68,7 +68,7 @@ func TestMiddleware(t *testing.T) {
 				RequestBody:  body,
 				StatusCode:   http.StatusOK,
 				ResponseTime: 1000,
-				ResponseSize: 18,
+				ResponseSize: 17,
```

Comment:
**Question**: what is the relevance of this change?
(Checked the issue and the patch release, but I might have overlooked this, because I haven't noticed it.)

Reply:
That is something I could not figure out yet (why is the actual `ResponseSize` changed) and I meant to ask around about it...

Reply:
Noted